webapp/components/admin_console/gitlab_settings.jsx

```diff
@@ -59,7 +59,7 @@ export default class GitLabSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.gitlab.enableTitle'
-                            defaultMessage='Enable Sign Up With GitLab: '
+                            defaultMessage='Enable account creation with GitLab: '
```

Comment:
Is this setting to enable both sign up and login with GitLab? maybe it should be "Enable authentication with GitLab" (0/5 on this)